packages/xr/src/lib/internal/events.ts

```diff
@@ -0,0 +1,30 @@
+export const events: Record<string, ((event: any) => void)[]> = {}
```

Comment:
As described with the navMeshes of `<TeleportControls>` as well, I don't know if we would run into conflicts if multiple XR-ready Threlte apps are mounted on a single page. To be on the safe side, we would probably add what are now globals (for instance `events`) to the context of the component `<XR>`.

Reply:
In that case the user won't experience any issues, these events that are being re-emitted originate [from the `window` object](https://developer.mozilla.org/en-US/docs/Web/API/XRSession/select_event) because only one XR session can exist on a single machine at any given time. Additionally, not binding these emitters to `<XR>`'s context allows for more flexibility when designing the scene. Let's say that I want to display a mesh as the "intro" for a VR experience before the user enters the session. I could do this:

```svelte
<XR />

<PointerControls>
  <T.Mesh on:pointerenter={doSomethingCool}>
     ...
  </T.Mesh>
</PointerControls>
```

The `TeleportControls` example in the docs does something similar. Still have concerns, or do you think these advantages are worth it?

Reply:
Alright, makes sense!